R/content.R

```diff
@@ -273,6 +273,56 @@ Content <- R6::R6Class(
       cat('content_item(client, guid = "', self$get_content()$guid, '")', "\n", sep = "")
       cat("\n")
       invisible(self)
+    },
+
+    #' @description Render or restart this content item.
```

Comment:
You don't have to do it in the docstring here, but it's one place to do it. What would it look like to actually use this?

```
connect <- client()
...
...$refresh()
```

Reply:
Yeah, the documentation definitely isn't complete.